client/src/actions/submission.js

```diff
@@ -0,0 +1,20 @@
+import { handleErrors } from './util';
+
+export function submitSolution(problemId, file) {
```

Comment:
Can you change this to `submitSubmission`? Because in the backend @rwongone and I only calling solution the teacher's correct submission.